helm/reana/templates/NOTES.txt

```diff
@@ -37,4 +37,11 @@ finalise its configuration.
     $ cd reana-demo-root6-roofit
     $ reana-client run -w demo
 
+If you already had REANA deployed and you are upgrading it, there is one step left to
```

Comment:
What about making it shorter as follows: 

```
If you are upgrading from a previous REANA release series, you may need to upgrade database schema now:

   $ kubectl -n {{ .Release.Namespace }} exec -i -t deployment/{{ .Release.Name }}-server -- reana-db alembic upgrade

Please see the dedicated `[upgrade](url)` documentation page with more information.

If you are deploying...
...
...
```

Reply:
I guess markdown links won't be very effective here. Should we leave it like `[upgrade](url)` regardless?

Reply:
Indeed, perhaps just use ASCII-friendly:

```
... please see the dedicated upgrade documentation at <http://example.org/foo/bar>.
```

Reply:
Also, please wrap the line so that it'll be <80 characters long. Easier to read on narrow terminals. (The command can remain longer, as the others...)